components/library/src/taxonomies/mod.rs

```diff
@@ -232,12 +235,15 @@ impl Taxonomy {
 
         // Check for taxon-specific template, or use generic as fallback.
         let specific_template = format!("{}/list.html", self.kind.name);
-        let template = match check_template_fallbacks(&specific_template, tera, &config.theme) {
-            Some(template) => template,
-            None => "taxonomy_list.html",
+        let template = if let Some(template) =
+            check_template_fallbacks(&specific_template, tera, &config.theme)
+        {
+            template
+        } else {
+            "taxonomy_list.html"
         };
```

Comment:
same

Reply:
checked it again and now clippy gives different suggestions. Not sure why, but I do agree that it's less good than a match. This time clippy suggested replacing it with `unwrap_or` which seems even better to me